packages/remix-cloudflare-pages/worker.ts

```diff
@@ -41,6 +41,11 @@ export function createFetchHandler<Env = any>({
   const handleFetch = async (context: EventContext<Env, any, any>) => {
     let response: Response | undefined;
 
+    const request = new Request(context.request);
+    // https://github.com/cloudflare/wrangler2/issues/117
```

Comment:
This indicates that the issue is during dev only. Should we do an env check here?

Reply:
It's a production issue as well. They are investigating and we should have this here for now regardless of local or prod.